src/components/reusable/table/table.stories.ts

```diff
@@ -259,7 +263,30 @@ export const Pagination: Story = {
         .pageSize=${args.pageSize}
         .pageNumber=${args.pageNumber}
         .pageSizeOptions=${args.pageSizeOptions}
-        .showPagination=${true}
+        showPagination
+      >
+      </story-table>
+    `;
+  },
+  parameters,
+};
+
+export const Legend: Story = {
+  args: {
+    pageSize: 5,
+    pageNumber: 1,
+    pageSizeOptions: [5, 10, 20, 30, 40, 50, 100],
+  },
+  render: (args) => {
+    return html`
+      <story-table
+        .tableTitle=${'Legend'}
+        .rows=${characters}
+        .pageSize=${args.pageSize}
```

Comment:
IMO **characters** should be more than 5 (at-least 10) to display pagination working since we are passing only pagination props here.